lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -102,17 +102,7 @@ If you need any further assistance then you can also [request help here](${
         config.productLinks!.help
       }) if you have any doubts and would like it reviewed.\n\n`
     );
-    const isConflicted = await scm.isBranchConflicted(
-      config.baseBranch!,
-      config.onboardingBranch!
-    );
-    if (isConflicted) {
-      configDesc += emojify(
-        `:warning: This PR has a merge conflict. However, Renovate is unable to automatically fix that due to edits in this branch. Please resolve the merge conflict manually.\n\n`
-      );
-    } else {
-      configDesc += `Important: Now that this branch is edited, Renovate can't rebase it from the base branch any more. If you make changes to the base branch that could impact this onboarding PR, please merge them manually.\n\n`;
-    }
+    configDesc += `Important: Now that this branch is edited, Renovate can't rebase it from the base branch any more. If you make changes to the base branch that could impact this onboarding PR, please merge them manually.\n\n`;
```

Comment:
According to your new changes, do we ever need to rebase from base branch automatically anymore?

Reply:
No probably, will test this. brb